src/neuroglancer/viewer.ts

```diff
@@ -208,7 +208,8 @@ export class Viewer extends RefCounted implements ViewerState {
   layerSpecification: TopLevelLayerListSpecification;
   layout: RootLayoutContainer;
 
-  jsonStateServer = new TrackableValue<string>('', validateStateServer);
+  stateServer = new TrackableValue<string>('', validateStateServer);
```

Comment:
I think this was a seunglab thing and could be removed

Reply:
Snuck in there somehow...